hovercontrol/src/hoverkite.rs

```diff
@@ -139,43 +160,57 @@ impl Hoverkite {
         side_command.write_to_std(port)?;
         Ok(())
     }
-}
 
-fn print_response(response: &Option<SideResponse>) {
-    match response {
-        Some(SideResponse {
-            side,
-            response: Response::Log(log),
-        }) => println!("{:?}: '{}'", side, log),
-        Some(SideResponse {
-            side,
-            response: Response::Position(position),
-        }) => println!("{:?} at {}", side, position),
-        Some(SideResponse {
-            side,
-            response:
-                Response::BatteryReadings {
-                    battery_voltage,
-                    backup_battery_voltage,
-                    motor_current,
-                },
-        }) => println!(
-            "{:?} battery voltage: {} mV, backup: {} mV, current {} mV",
-            side, battery_voltage, backup_battery_voltage, motor_current
-        ),
-        Some(SideResponse {
-            side,
-            response: Response::ChargeState { charger_connected },
-        }) => println!(
-            "{:?} {}",
-            side,
-            if *charger_connected {
-                "charger connected"
-            } else {
-                "charger not connected"
+    fn print_response(&self, response: &Option<SideResponse>) {
```

Comment:
Move this into a separate module that calls into Hoverkite

Maybe Hoverkite gets moved into hoverkite_protocol under a feature flag, or into its own crate?